AGE_65_PLUS.qmd

```diff
@@ -48,23 +48,38 @@ total_df$value = total_df$value - df$value
 
 state_choropleth(percent_df,
                  title = "PERCENT POPULATION OVER 65 \n2021 1-YEAR ACS S0101_C01_001",
-                 legend = "PERCENTAGE"
+                 legend = "PERCENTAGE",
+                 num_colors = 4
                  )
 
-ggplot(percent_df, aes(y = value)) +
+x = ggplot(percent_df, aes(y = value)) +
   geom_boxplot(color="red", fill="orange", alpha=1) + 
   scale_y_continuous(label=comma) +
   ggtitle("PERCENT DISTRIBUTION STATES\n WITH POPULATION 65+") +
   xlab("")+
 ylab("POPULATION")+
   theme(axis.text.x=element_blank(),
         axis.ticks.x=element_blank()) 
+ggplotly(x)
+
```

Comment:
Please delete the whitespace between the last line of code hear and the trailing ```